host/3.0/bionic/arm32v7/dotnet/dotnet.Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN EXTENSION_BUNDLE_VERSION=1.8.1 && \
     mkdir -p /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION_V2 && \
     unzip /$EXTENSION_BUNDLE_FILENAME_V2 -d /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION_V2 && \
     rm -f /$EXTENSION_BUNDLE_FILENAME_V2 &&\
-    EXTENSION_BUNDLE_VERSION_V3=3.20.0 && \
```

Comment:
Add comment - link to https://github.com/Azure/azure-functions-host/commit/d251aef1d33a585294a849e6750c4a950d75fc1b and call out do not upgrade bundles

Reply:
I added a note above the extension bundles.